openapi.yaml

```diff
@@ -33886,30 +33886,15 @@ tags:
   - name: Transit Gateways
   - name: Provider Accounts
   - name: Custom Domains
-  - name: Resource Quotas
```

Comment:
@mheap I couldn't pinpoint the cause for removal of these tags, wanted to check with you.

Reply:
I fixed a bug last week in https://github.com/Kong/platform-api/pull/971/commits/26bd9cf5c3e0fda3713116e804409975fc1c9623